README.md

```diff
@@ -28,7 +34,16 @@ Figure-1 is the [Architecture and Working Scope](https://github.com/intel/intel-
 
 This [section](/docs/supported_platforms.md) describes the RHOCP infrastructure and Intel hardware features supported by this project. The project lifecycle and support channels can also be found [here](docs/supported_platforms.md#support). 
 
-# Getting started  
+# Getting started
+
+## BIOS Configuration
```

Comment:
I think we ever plan to put the BIOS configuration in the Provisioning RHOCP cluster section.

Reply:
I think it makes more sense here because we do BIOS configuration before provisioning or setting up cluster. BIOS configuration on each node and then provision cluster.